### PR TITLE
Install streams_explorer package after dependencies

### PR DESCRIPTION
**Pull request by disrupted**
This fixes an issue introduced by #220, where the Docker image would fail to launch because importlib can't find the `streams_explorer` package. This is now fixed by installing the actual package through pip rather than just copying the module folder into the Docker image.
Once poetry 1.2.0 is released we might be able to transition to `poetry add -e`, which is a new feature currently in pre-release.

Additionally this gets rid of a workaround which was used to copy the README from repository root to the Python package before release. Instead, it is now located there and symlinked to root. The presentation on GitHub is unaffected by this change.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -25,6 +25,9 @@ RUN pip install -U pip poetry && \
     poetry install --no-dev --no-interaction
 COPY ./backend /app
 
+# install streams_explorer package
+RUN pip install -e .
```
Comment (philipp94831) [inline Dockerfile:29]: This was not needed before?

Comment (disrupted) [reply Dockerfile:29]: yes, this is only needed now because we're reading the package version through importlib. (as noted in PR description)

**Comment by VictorKuenstler** [comment]
Maybe we should think about handling the Readme for Python package as a separate Readme because the content of the overall Readme might be confusing. 🤔 But, of course, that would be a separate PR.

**Review by VictorKuenstler — APPROVED** [review]
LGTM

**Comment by disrupted** [comment]
> Maybe we should think about handling the Readme for Python package as a separate Readme because the content of the overall Readme might be confusing. 🤔 But, of course, that would be a separate PR.

good point! Then again, the Python package is probably always used with the frontend. We can think of slimming it down somewhat for PyPI.

**Comment by VictorKuenstler** [comment]
> good point! Then again, the Python package is probably always used with the frontend. We can think of slimming it down somewhat for PyPI.

Yep exactly. Highlighting the purpose of the Python Package could be good. This could make it easier to understand why and when to use it.

**Pull request merged**